### ISSUE TITLE
Since Java 11 Update 9 is not neccesary Automatic-Module-Name in not-aggregated maven-javadoc-plugin execution

### ISSUE THREAD

**Issue by antoniosanct** [rometools/rome#517]
Related:

https://stackoverflow.com/questions/62206112/maven-javadoc-plugin-how-to-update-the-module-path-dynamically

**Comment by PatrickGotthard** [comment]
Hi @antoniosanct,

could you please summarize what you want to achieve with this issue? I really have no idea what you want even when reading the SO thread.

Regards,
Patrick

**Comment by antoniosanct** [comment]
Hi, @PatrickGotthard:

Sorry for not explaining the issue correctly.

While I was fixing the Javadocs bugs and testing the maven goals, using javadoc:javadoc I detected an error in javadoc.exe (jdk 11 update 7) because of rome-test and rome-opml are unnamed modules. They haven't Automatic-Module-Name tag in pom.xml declaration @ maven-jar-plugin configuration, and the other modules they have. However, including these tags don't fix the javadoc error.

I understood that it would be a javac/javadoc bug! I fixed in my own fork repo deleting these tags and updating with jdk 11 update 13. The error dissappeared.

It is not a issue at all, but it is interesting that is mandatory for Rome use since JDK 11 update 9 to execute successfully the totally POM goals (including javadoc with <doclint>all</doclint>). But that's just my opinion :)

Regards!
Antonio
